### PR TITLE
Digital clock

### PR DESCRIPTION
**Pull request by DragonUncaged**
#26 
I am SWOC participant.

### CODE REVIEW

**Review by Ovenoboyo — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Review by nandini7637 — CHANGES_REQUESTED** [review]
There is issue with responsiveness, please make requires changes
image

**Review by saikatmu31 — APPROVED** [review]

**Review by Abirpal202049 — APPROVED** [review]

**Review by Abirpal202049 — CHANGES_REQUESTED** [review]
Make it responsive

**Review by saikatmu31 — CHANGES_REQUESTED** [review]
Improve the Responsiveness by Using media query

**Comment by Abirpal202049** [comment]
@DragonUncaged put your files inside proper folder

File: Clock/Digital_Clock/code/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+function clock(){
+    let hour = document.getElementById("hour");
+    let minute = document.getElementById("minute");
+    let seconds = document.getElementById("seconds");
+    let ampm = document.getElementById("ampm");
+
+    let h = new Date().getHours();
+    let m = new Date().getMinutes();
+    let s = new Date().getSeconds();
+    var am = "AM";
```
Comment (Ovenoboyo) [inline Clock/Digital_Clock/code/index.js:10]: Prefer using "const" instead of "let" wherever possible

File: Clock/Digital_Clock/code/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        var am = "PM";
+    }
+
+    hour.innerHTML = h;
+    minute.innerHTML = m;
+    seconds.innerHTML = s;
+    ampm.innerHTML = am;
+    
+}
+var interval = setInterval(clock,1000)
```
Comment (Ovenoboyo) [inline Clock/Digital_Clock/code/index.js:22]: Avoid using var altogether. Instead use let or const

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

**Comment by nandini7637** [comment]
@DragonUncaged have you worked on the changes?

**Review by saikatmu31 — APPROVED** [review]

**Comment by nandini7637** [comment]
@DragonUncaged  updates? 
Should I close this PR?

**Comment by DragonUncaged** [comment]
> @DragonUncaged updates? Should I close this PR?

NO

**Comment by nandini7637** [comment]
> > @DragonUncaged updates? Should I close this PR?
> 
> NO I will be doing it

Deadline for this is Saturday

**Comment by DragonUncaged** [comment]
Digital-clock-demoo

- [x] Changes Regarding the responsiveness
Made as much responsive as i can with retaining the beauty of the clock
- [x] Changes Regarding the JavaScript variable
 cannot use const everywhere but tried to use where ever possible.

**Review by Abirpal202049 — APPROVED** [review]

**Review by thepranaygupta — APPROVED** [review]

**Pull request merged**